src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: start;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  display: block;
+  background: url(images/star.svg) no-repeat center center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, sans-serif;
+
+  --card-text: #060b35;
+  --card-additional-text: #616070;
+  --card-main-color: #00acdc;
+}
+
+.card__container {
+  padding: 16px;
+}
+
+.card__featured-image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 40px;
```

Comment:
10 point from [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)
 Be consistent with your margins. Add only top or bottom, don't add both.
 40px margin move to .card-title margin-top, as other card elements